components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -37,6 +37,7 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
   public modifyID = false; // Whether the edit ID form is open or not.
   public conflictError = false;
   public addPolicies = true;
+  public addTeams = true;
```

Comment:
Hmm is `create-object-modal` still the right name? It seems quite specific to have policy- and team-creation-related code in here 🤔

Reply:
`s/pattern we've established/hack/`  😉

Reply:
Anyways, it's fine! 😃